### PR TITLE
fix(packages): fix offline packages config

### PR DESCRIPTION
**Pull request by wuhuizuo**
add tiproxy for from v7.6.0

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: packages/offline-packages.yaml.tmpl [hunk 1]
```diff
@@ -52,8 +52,106 @@ editions:
...
+              - { name: alertmanager, src: { type: tiup-clone, version: latest } }
+              - { name: blackbox_exporter, src: { type: tiup-clone, version: latest } }
+              - { name: node_exporter, src: { type: tiup-clone, version: latest } }
+              - { name: tiup, src: { type: tiup-clone, version: latest } }
+              - { name: cluster, src: { type: tiup-clone, version: latest } }
+              - { name: insight, src: { type: tiup-clone, version: latest } }
+              - { name: diag, src: { type: tiup-clone, version: latest } }
+              - { name: influxdb, src: { type: tiup-clone, version: latest } }
+              - { name: playground, src: { type: tiup-clone, version: latest } }
+              - { name: tiproxy, src: { type: tiup-clone, version: latest" } }
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:74]: comparing with v7.5.0, it added `tiproxy`.

File: packages/offline-packages.yaml.tmpl [hunk 2]
```diff
@@ -260,9 +358,142 @@ editions:
...
+              - { name: alertmanager, src: { type: tiup-clone, version: latest } }
+              - { name: blackbox_exporter, src: { type: tiup-clone, version: latest } }
+              - { name: node_exporter, src: { type: tiup-clone, version: latest } }
+              - { name: tiup, src: { type: tiup-clone, version: latest } }
+              - { name: cluster, src: { type: tiup-clone, version: latest } }
+              - { name: insight, src: { type: tiup-clone, version: latest } }
+              - { name: diag, src: { type: tiup-clone, version: latest } }
+              - { name: influxdb, src: { type: tiup-clone, version: latest } }
+              - { name: playground, src: { type: tiup-clone, version: latest } }
+              - { name: tiproxy, src: { type: tiup-clone, version: latest } }
```
Comment (wuhuizuo) [inline packages/offline-packages.yaml.tmpl:381]: comparing with v7.5.0, it added `tiproxy`.

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**